packages/suite-desktop-core/e2e/tests/analytics/events.test.ts

```
@@ -0,0 +1,184 @@
+        >(EventType.TransportType);
+        expect(transportTypeEvent.type).toBe('BridgeTransport');
+        expect(parseInt(transportTypeEvent.version, 10)).not.toBeNaN();
+
+        // device-disconnect is logged 4th
+        await trezorUserEnvLink.stopEmu();
+
+        await expect.poll(() => analytics.requests).toHaveLength(4); // Poll to prevent race condition
+        expect(analytics.requests[3]).toHaveProperty('c_type', EventType.DeviceDisconnect);
+    });
+
+    test.only('reports suite-ready after enabling analytics on app initial run', async ({
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)